AltStore/Resources/Assets.xcassets/Image.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "180.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "180 1.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "180 2.png",
```

Comment:
What’s up with the naming of these files? If they’re different sizes, why not name them like SideStore-1x.png? And why is the imageset just called “Image”?